code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -177,3 +177,38 @@
 			M.apply_damage(ranged_damage, WHITE_DAMAGE, null, M.run_armor_check(null, WHITE_DAMAGE), spread_damage = TRUE)
 			new /obj/effect/temp_visual/remorse(get_turf(M))
 		targets = list()
+
+/obj/item/ego_weapon/thirteen
+	name = "for whom the bell tolls"
+	desc = "There is nothing else than now. There is neither yesterday, certainly, nor is there any tomorrow."
+	special = "This weapon deals bonus damage on the 13th hit."
+	icon_state = "thirteen"
+	force = 35
+	damtype = PALE_DAMAGE
+	armortype = PALE_DAMAGE
+	attack_verb_continuous = list("cuts", "attacks", "slashes")
+	attack_verb_simple = list("cut", "attack", "slash")
+	hitsound = 'sound/weapons/rapierhit.ogg'
+	attribute_requirements = list(
+							JUSTICE_ATTRIBUTE = 60
+							)
+	var/combo = 0
+	var/combo_time
+	var/combo_wait = 3 SECONDS
+
+//On the 13th hit, Deals user justice x 3
+/obj/item/ego_weapon/thirteen/attack(mob/living/M, mob/living/user)
+	if(!CanUseEgo(user))
+		return
+	if(world.time > combo_time)
+		combo = 0
+	combo_time = world.time + combo_wait
+	if(combo==13)
+		combo = 0
+		var/userjust = (get_attribute_level(user, JUSTICE_ATTRIBUTE))
+		force = userjust*3
```

Comment:
~820 damage at max justice.

Reply:
Actually approx 390, and you have to hit 13 times in a row, and after a bit of time it resets. I'll reduce it to double.